src/Dropdown/Dropdown.tsx

```diff
@@ -116,7 +124,10 @@ export default <T,>({
                 {items.map(item => (
                     <button
                         type="button"
-                        className="tw-bg-transparent tw-clear-both tw-block tw-h-6 tw-w-full tw-whitespace-nowrap tw-border-0 tw-px-2 tw-py-1 tw-text-left tw-font-normal tw-text-white hover:tw-bg-gray-600 focus:tw-bg-gray-600"
+                        className={classNames(
```

Comment:
Given that the dropdown items are also smaller you also need to look at 
`
  data-height={
                    numItemsBeforeScroll > 0 &&
                    items.length > numItemsBeforeScroll
                }
`

to make sure that the correct about of item are shown when in sm config